eduVPN/ConfigurationService.swift

```diff
@@ -211,6 +197,76 @@ class ConfigurationService {
         }
     }
     
+    private func forgetKeyPair(certificateCommonName: String) {
+        try? keychainService.removeIdentity(for: certificateCommonName)
+        
+        var keyPairs = UserDefaults.standard.array(forKey: "keyPairs") ?? []
```

Comment:
`keyPars` is used a few times; maybe make it a `private let keyPairsKey = "keyPairs"` ?